UPGRADE-1.8.md

```diff
@@ -119,6 +119,9 @@ Some translations have changed, you may want to search for them in your project:
 - `sylius.email.shipment_confirmation.you_can_check_its_location` has been removed.
 - `sylius.email.shipment_confirmation.you_can_check_its_location_with_the_tracking_code` has been added instead of the two above.
 
+### Deprecations
+Service `Sylius\Component\Core\Calculator\ProductVariantPriceCalculatorInterface` has been deprecated. Use `Sylius\Component\Core\Calculator\ProductVariantPricesCalculatorInterface` instead.
```

Comment:
This note does not make sense on the 1.11 branch, should be removed from this PR

Reply:
Ah, you're right 😞